packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -52,22 +50,22 @@ void main() {
 
     await tester.pumpWidget(buildFrame(null));
     await tester.pumpAndSettle();
-    expect(getCheckboxListTileRenderer(), paints..path(color: const Color(0xFFFFFFFF))); // paints's color is 0xFFFFFFFF (default color)
+    expect(getCheckboxListTileRenderer(), paints..path(color: const Color(0xFFFFFFFF))); // paints' color is 0xFFFFFFFF (default color)
```

Comment:
I think you can just remove the comments here and below: they don't add anything.

Reply:
Indeed. Not sure how I caught the typo, on these lines but missed it below. Anyway I have removed them. Thx.